compiler/resolution/functionResolution.cpp

```diff
@@ -11247,6 +11247,20 @@ static void resolveExterns()
   }
 }
 
+// Returns 'true' if the type 'at' was resolved.
+static void maybeForceResolveAggregateType(AggregateType* at) {
```

Comment:
This comment seems stale already.

Reply:
It most definitely is, how could I forget to delete it?! 😁